ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -1127,6 +1168,24 @@
 /* End PBXHeadersBuildPhase section */
 
 /* Begin PBXNativeTarget section */
+		60967E071D76B56D00863853 /* ADAL Preprocessing */ = {
```

Comment:
Now it works by running script for each target :)

I find that I should actually set the version number in ${BUILT_PRODUCTS_DIR}/${INFOPLIST_PATH}.